samps/contests/mini/config/contest.yaml

```diff
@@ -35,6 +35,16 @@ default-validator: /home/judge/cdp/config/validefault {:infile} {:outfile} {:ans
 # Load data files into pc2 as SerializedFiles, not using CDP external files
 load-data-files: false
 
+# Loader will automatically load accounts file CDPDIRconfig/accounts_load.tsv
+
+# To load a different load accounts file use key load-accounts-file 
+# load file fron ocnfig/ dir 

```

Comment:
Typo in comment

Reply:
in the future please state the typo, rather than requiring the developer to do a best guess

Reply:
fixed 3 typos, pushed fix

Reply:
> in the future please state the typo, rather than requiring the developer to do a best guess

The comment was placed on the line containing the typo, which line contains only five words.  I didn't think it was too onerous a job to read that line and see the (obvious) spelling error.   But sure, I'll try to remember to literally quote the exact error in the future.